### PR TITLE
Add drinking wine option

### PR DESCRIPTION
**Pull request by jhakimyanova**

### CODE REVIEW

**Comment by jhakimyanova** [comment]
Please review this, @casey-collab

**Comment by pmallory** [comment]
@casey-collab Can you review this PR please?

File: english/wine/wine.md [hunk 1]
```diff
@@ -0,0 +1 @@
+The glass of wine is the best thing that happened with you during this day.
```
Comment (casey-collab) [inline english/wine/wine.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by jhakimyanova** [comment]
Please review this again, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :smiley_cat:

**Pull request merged**